### DISCUSSION TITLE
system crashed after upgrade 28/03/2022

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#60] [General] [answered]
Helo. After yesterday upgrade system crashed. No auttenticated manager. No libstgc++6.so library. Polkit service did not run.
When it will be safe to upgrade system via luet?

**Comment by joostruis** [comment, marked as answer]
The problem was noticed and caused by a config file that was protected but should be merged automatically. 
We solved the issues in this commit: https://github.com/mocaccinoOS/desktop/commit/75201b5dc80594a2de868d11de817896c0ef9dce

If you are affected by this you should run:

`mos config-update update --interactive=false -p /etc/ld.so.conf.d`

and then:
`source /etc/profile && env-update`
